### DISCUSSION TITLE
Add Zoom to 16:9 option

### DISCUSSION THREAD

**Discussion by jasikpark** [rxhanson/Rectangle#392] [Ideas]
Something that would be helpful is the option to maximize a window to a particular aspect ratio, namely 16:9 for screen capture options. Since macbook pro displays are 16:10, but generally something like a Twitch stream should be some form of 16:9, I'm trying to find a window manager that can do this for me.

I'm a long time user of Rectangle, I love it! I jumped over here at the reccomendation of S_____ the similar window manager that stopped being developed?

Thanks!

**Comment by rxhanson** [comment]
Thanks for the kind words! At the moment, I don't have plans to add more/custom window sizes to Rectangle. 

My paid superset of Rectangle, Hookshot (https://hookshot.app) does have a custom shortcut creator that can handle any window dimension, either in absolute dimensions or in screen percentage. Here's a post I wrote about the custom shortcut creator that talks about why this is not in Rectangle and how to use it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24
